### ISSUE TITLE
Resetting coqtop after it dies badly removes breakpoints from the CoqIDE visual debugger

### ISSUE THREAD

**Issue by JasonGross** [coq/coq#14332]
If I press the interrupt button twice as in https://github.com/coq/coq/issues/14331, when I click "reset" on coqtop died badly, all breakpoints are removed.  Futhermore, if I try to set a new breakpoint in the same file without restarting coqtop, I get "coqtop died badly" immediately:
![image](https://user-images.githubusercontent.com/396076/118490758-4456ac00-b6ec-11eb-9a11-e23e8094ba9b.png)

**Comment by jfehrle** [comment]
For now, try to avoid pressing "interrupt" twice, wait for the first one to return.

**Comment by JasonGross** [comment]
Indeed, waiting a while fixes the problem.  So this is low priority

**Comment by JasonGross** [comment]
Ah, oops, sorry, wrong issue to comment on
